<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPresidential</class>
 <widget class="QWidget" name="addPresidential">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Başkanlık Ekle</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>210</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Başkanlık ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Başkanlık Adı:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Deniz Adı:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Şehir:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtPId"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSeaName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCity"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>İptal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
